src/templates/default/index.html.twig

```diff
@@ -1,5 +1,9 @@
 {% extends 'base.html.twig'|extendable(event_type) %}
 
+{% block head %}
+    {{ getCSS('default') }}
```

Comment:
tu si zahodil extendabilitu, nie?

Reply:
Hej, problem je, ze som nevedel ako by som si mohol vytiahnut z miesta kde sa to vola meno tej templaty. Jedine, ze je tam aj zadana.

Reply:
Nie uplne chapem tomu co si napisal. Skus si pozriet ako to riesime v `|extendable(event_type)` a ci by nam to tu nepomohlo, podla mna jednoducha modifikacia toho by to ten problem vyriesila.